src/lib/queries/lexical-query.js

```diff
@@ -187,6 +187,14 @@ export default class LexicalQuery extends Query {
     // short definitions provided by the maAdapter
     if (lexiconShortOpts.allow) {
       this.homonym.lexemes.forEach((l) => { l.meaning.clearShortDefs() })
+    } else {
+      // we won't have any remaining valid short definition requests
+      // so go ahead and publish the SHORT_DEFS_READY event so that the UI
+      // can update itself
+      LexicalQuery.evt.SHORT_DEFS_READY.pub({
+        requestType: 'short',
+        homonym: this.homonym
```

Comment:
We probably need to provide one more prop: `word`, which should be a lexeme.lemma.word. It will be used as a `lemma` parameter in a message string: `"{requestType} request is completed successfully. Lemma: \"{lemma}\""`. 
I'm not sure if this message will be ever shown in this particular case, but I think it's better to play safe.

Reply:
oh, great catch, thank you! Will fix.

Reply:
(although actually, I think this is probably not working right for the other publications of this event, since the client adapter never sets this :-( but I will fix it for this instance and enter an issue for the  client-adapters)